src/Shared/Microsoft.DotNet.Scaffolding.Shared/ConsoleLogger.cs

```diff
@@ -8,8 +8,11 @@ internal class ConsoleLogger : IConsoleLogger
         private readonly bool _jsonOutput;
         private readonly bool _silent;
 
-        public ConsoleLogger(bool jsonOutput = false, bool silent = false)
+        private string CommandName { get; }
+
+        public ConsoleLogger(string commandName = null, bool jsonOutput = false, bool silent = false)
```

Comment:
since `commandName` can be null, new JsonResponse serialization can error out I think, maybe let's make default to `string.Empty`

Reply:
Done 👍🏻